src/webview/MessageList.js

```diff
@@ -105,34 +105,32 @@ export type Props = {|
 |};
 
 /**
- * The path to the webview assets, represented as a `file:`-scheme URL.
+ * The path to the webview assets, represented as a `file:`-schema URL. This may
+ * be a relative URL; if so, it is relative to the bundle or package root, and
+ * the scheme is necessarily omitted as implicit. [1]
  *
- * This may be a relative URL, relative to whatever base URL a `WebView`
- * component will use to interpret its own `source.baseUrl` prop.  (If
- * relative, the scheme naturally is left implicit.)
+ * Because this URL may be relative, it shouldn't be used for URLs in the
+ * browser environment _within_ the `WebView`; to reach the webview assets
+ * there, just use `./`.
  *
- * See the `tools/build-webview` script for more details.
+ * [1] Unlike some other URL schemas (notably `https:`), `file:` URLs don't
+ *     support use of an explicit schema with a relative path.
```

Comment:
Huh, does `https:` really support this? I'm not finding it [in the spec](https://url.spec.whatwg.org/#url-writing), or in some brief experimentation.

Reply:
> ... but that seems to be because validation errors don't actually terminate URL parsing. (See also line 5 in the table at [the top of that section](https://url.spec.whatwg.org/#urls).)

Hmm! Good to know.